templates/payment/form-installment.php

```diff
@@ -1,49 +1,9 @@
 <?php if ( ! empty( $viewData['installment_backends'] ) ) : ?>
-	<fieldset id="omise-form-installment">
-		<ul class="omise-banks-list">
-			<?php foreach ( $viewData['installment_backends'] as $backend ) : ?>
-				<li class="item">
-					<input id="<?php echo $backend->_id; ?>" type="radio" name="source[type]" value="<?php echo $backend->_id; ?>" />
-					<label for="<?php echo $backend->_id; ?>">
-						<div class="bank-logo <?php echo $backend->provider_code; ?>"></div>
-						<div class="bank-label">
-							<span class="title"><?php echo $backend->provider_name; ?></span><br/>
-							<select id="<?php echo $backend->_id; ?>_installment_terms" name="<?php echo $backend->_id; ?>_installment_terms" class="installment-term-select-box">
-								<option>Select term</option>
-								<?php foreach ( $backend->available_plans as $installment_plan ) : ?>
-									<option value="<?php echo $installment_plan['term_length']; ?>">
-										<?php
-										echo sprintf(
-											__( '%d months', 'omise' ),
-											$installment_plan['term_length']
-										);
-										?>
-
-										<?php
-										echo sprintf(
-											__( '( %s / months )', 'omise' ),
-											wc_price( $installment_plan['monthly_amount'] )
-										);
-										?>
-									</option>
-								<?php endforeach; ?>
-							</select>
-							<?php if ( ! $viewData['is_zero_interest'] ): ?>
-								<br/><span class="omise-installment-interest-rate">
-									<?php echo sprintf( __( '( interest %g%% )', 'omise' ), $backend->interest_rate ); ?>
-								</span>
-							<?php endif; ?>
-						</div>
-					</label>
-				</li>
-			<?php endforeach; ?>
-		</ul>
-		<div class="omise-buttom-note">
-			<p>
-				<?php echo $viewData['is_zero_interest'] ? __( 'All installment payments are interest free', 'omise' ) : __( 'Monthly payment rates shown may be inaccurate as interest rates are subject to change by its bank issuer.', 'omise' ); ?>
-			</p>
-		</div>
-	</fieldset>
+	<div id="omise-installment" style="width:100%; max-width: 400px;"></div>
+	<script>
+		window.LOCALE = `<?php echo get_locale(); ?>`;
+		window.OMISE_CUSTOM_FONT_OTHER = 'Other';
+	</script>
```

Comment:
By removing the old form, the current installment payment will not be visible. We still need to support the old installment.

Reply:
Hmm, But it will be rendered by pay.js, which we have decided to use either WLB or non-WLB internally.
For account that don't have whitelabel installment enabled, will see the old installment with the new ui.

Reply:
Ah okay. I didn't realize that.

Reply:
Ah okay. That will work.